internal/credentials.go

```diff
@@ -28,9 +28,9 @@ func (cr *Credentials) separator() byte {
 
 func (cr *Credentials) locations() []string {
 	return []string{
+		"/run/secrets/SCCcredentials",
```

Comment:
I'm not sure it's a good idea to put it above `/etc`. Sorry to be a pain. :wink: